go.mod

```diff
@@ -7,10 +7,10 @@ replace github.com/ipfs/go-verifcid => github.com/celestiaorg/go-verifcid v0.0.1
 require (
 	cosmossdk.io/errors v1.0.0-beta.7
 	cosmossdk.io/math v1.0.0-rc.0
-	github.com/BurntSushi/toml v1.3.0
+	github.com/BurntSushi/toml v1.3.2
 	github.com/alecthomas/jsonschema v0.0.0-20200530073317-71f438968921
 	github.com/benbjohnson/clock v1.3.5
-	github.com/celestiaorg/celestia-app v1.0.0-rc5
+	github.com/celestiaorg/celestia-app v1.0.0-rc7.0.20230706110617-0f1ce2ce52e7
```

Comment:
we still need to swap to the official tag once release is cut

Reply:
is this PR mergable? if it's still waiting for an app rc, make it a draft to avoid confusion :)

Reply:
yeah sorry we can keep it a draft, as of ~30minutes ago this was updated